### ISSUE TITLE
Add support for temperature_probe type temperature sensor

### ISSUE THREAD

**Issue by hannut** [marcolivierarsenault/moonraker-home-assistant#449]
## Is your feature request related to a problem? Please describe.

Eddy current bed probe uses **temperature_probe** type entry in the configuration. It shows on the Mainsail UI with the other temperatures. It would be useful to be able to log this temperature with Home Assistant.

```
# This section is only relevant for Eddy USB. Comment it out for Eddy Coil.
[temperature_probe btt_eddy]
sensor_type: Generic 3950
sensor_pin: eddy:gpio26
horizontal_move_z: 2
```

## Describe your setup

Please describe your Printer setup, (especially software stack)
Klipper, Moonraker, Mainsail
BTT CB1, BTT Eddy USB

## Describe the solution you'd like

have a temperature sensor for temperature_probe in HA

**Comment by marcolivierarsenault** [comment]
I think I have a quick fix for this but I would like if you could test it. 

Can you go in the source code on your Home Assistant deployment and add 1 line of code in `/custom_components/moonraker/sensor.py`

around the line 332 you should see something like 

```python
    temperature_keys = [
        "temperature_sensor",
        "temperature_fan",
        "tmc2240",
        "bme280",
        "htu21d",
        "lm75",
    ]
```

Can you add your `temperature_probe` in that list so it looks like 

```python
    temperature_keys = [
        "temperature_sensor",
        "temperature_fan",
        "temperature_probe",
        "tmc2240",
        "bme280",
        "htu21d",
        "lm75",
    ]
```

then restart home assistant and let me know if it works

**Comment by hannut** [comment]
Hi,

this works. however there is unnecessary precision on the sensor 😄 

> trident Btt Eddy Temp
> 21 seconds ago
> 48,8259379346336 °C

**Comment by marcolivierarsenault** [comment]
Shipped in Version 1.5.0
